src/components/layout/index.tsx

```diff
@@ -4,30 +4,57 @@ import CookiesBanner from "@/components/cookies";
 import { useEffect, useState } from "react";
 import Head from "next/head";
 import { urlFor } from "../../../util/client";
+import GoogleAnalytics from "../google-analytics";
+import { getLocalStorage } from "../../../util/localStorageHelper";
 
 export default function Layout({ children }: { children: React.ReactNode }) {
   const [isShowCookie, setIsShowCookie] = useState(true);
+  const [isConsentCookie, setIsConsentCookie] = useState(false);
   const [favicon, setFavicon] = useState("");
+  const [ga, setGa] = useState("");
+
+  const enviroment = process.env.NEXT_PUBLIC_ENVIROMENT;
```

Comment:
Can the spelling change to environment instead of enviroment please 😊 In the const variable and the .env

Reply:
Thanks Maddie, it's done :)